src/core/main.c

```diff
@@ -92,13 +94,19 @@ int main(int argc, char *argv[]) {
 	}
 
 	struct xdpw_state state = {
+		.xdpw_sessions = (struct wl_list) { 0 },
```

Comment:
No need to zero-initialize this field. Omitted fields are already zero-initialized.